src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -166,6 +166,9 @@ export class ListingCreateComponent implements OnInit {
         }
         if (this.editMode) {
             this.afterEdit();
+        } else {
```

Comment:
what if it's not confirmed? for example Rolledback

Reply:
Added condition to keep form values if rollback is chosen